applications/vanilla/models/class.categorymodel.php

```diff
@@ -153,25 +166,93 @@ private static function loadAllCategories() {
     }
 
     /**
-     * @param $Category
+     * Calculate the user-specific information on a category.
+     *
+     * @param array &$category The category to calculate.
      */
-    private static function calculateUser(&$Category) {
-        $Category['Url'] = url($Category['Url'], '//');
-        if ($Photo = val('Photo', $Category)) {
-            $Category['PhotoUrl'] = Gdn_Upload::url($Photo);
+    private function calculateUser(&$category, $addUserCategory = null) {
+        $category['Url'] = url($category['Url'], '//');
```

Comment:
This is actually not a change in functionality. I think the URL is calculated as a path and then this switches it to a proper URL.